app/controllers/reports/regions_controller.rb

```diff
@@ -129,6 +129,13 @@ def diabetes
       raise ActionController::RoutingError.new("Not Found")
     end
 
+    start_period = @period.advance(months: -(Reports::MAX_MONTHS_OF_DATA - 1))
+    range = Range.new(start_period, @period)
+    @repository = Reports::Repository.new(@region, periods: range)
+    @presenter = Reports::RepositoryPresenter.new(@repository)
+    @data = @presenter.call(@region)
+    @with_ltfu = with_ltfu?
```

Comment:
Should we move this to a `before_action` since it's common between `show`?

Reply:
Yes, will do this as a part of a seperate refactoring PR.